plat/hisilicon/hikey960/hi3660_mailbox.c

```diff
@@ -0,0 +1,166 @@
+/*
```

Comment:
@hzhuang1 
Static analysis tool indicated that this file is not used - is this intentional? If not used / needed, please could it be removed?